fcgl/madrid-native
# Implement search

## Problem

We need to have product search capabilities

## Solution
Implement the API that the search team worked on 

## Test
1. Run the search repo
2. Make a search for: coca, cola, Coca Cola, banana
    * They should all appear
3. Make a search for anything else
    * You should get an "no results found" message
